### DISCUSSION TITLE
Reduce size of png

### DISCUSSION THREAD

**Discussion by pepeu93** [ImageMagick/ImageMagick#4764] [Help]
What best command to reduce the size of a png without losing quality or changing its dimension?

**Comment by fmw42** [comment]
pngcrush  see https://legacy.imagemagick.org/Usage/formats/#png_non-im

**Reply by pepeu93** [reply]
@fmw42 using ImageMagick, please.

**Reply by fmw42** [reply]
ImageMagick controls are not very good.  But see -quality at https://imagemagick.org/script/command-line-options.php#quality and https://legacy.imagemagick.org/Usage/formats/#png_write

**Comment by snibgo** [comment]
pngcrush can try combinations of PNG parameters to find which gives the smallest size.

IM doesn't have that ability built-in. But a simple script can step try all the "-quality" numbers 0 to 100 to find the smallest output.

**Reply by pepeu93** [reply]
I see, whats the command using pngcrush? Im trying to reduce the size of a png without losing quality or changing its dimension.

**Reply by fmw42** [reply]
Sorry, I do not know.

**Reply by pepeu93** [reply]
I remember using one command in ImageMagick to it, but i could not find it anymore.
I was comparing the size compressing using the website TnyPic and IM, and it was very similar.

**Comment by snibgo** [comment]
> I see, whats the command using pngcrush?

"pngcrush -help"
